### PR TITLE
Adds support for inbloombakery

### LINKED ISSUE #1053: inbloombakery support
Please check that recipes published on the website you're requesting are public (we can't currently scrape recipes that require an account login), and add sample recipe URL(s) below:

- https://inbloombakery.com/the-best-cinnamon-rolls-ever/

### PR DESCRIPTION
**Pull request by jknndy**
Resolves #1053

### CODE REVIEW

**Review by jayaddison — APPROVED** [review]
Looks good to me :+1: 

Something to try not to forget: this site seems to have both `ratings` and a `ratings_count` (ref: #1061).

File: tests/test_data/inbloombakery.com/inbloombakery.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        "1/4 tsp salt",
+        "1/4 cup (60 ml) heavy cream (for pouring in between the rolls)",
+        "6 tbsp (84 g) unsalted butter, very softened",
+        "6 oz (170 g) cream cheese, cold",
+        "3/4 cup (97 g) powdered sugar",
+        "1/2 tbsp vanilla"
+      ],
+      "purpose": null
+    }
+  ],
```
Comment (jayaddison) [inline tests/test_data/inbloombakery.com/inbloombakery.json:39]: Ah, one question here: do we need test coverage on ingredient groups when a scraper does not implement specific support for them?  I think it'd probably be OK to remove the `ingredient_groups` field from the test expectations.

Comment (jknndy) [reply tests/test_data/inbloombakery.com/inbloombakery.json:39]: I definitely agree, right now taking them out returns an error but it's worth looking into. Still good to merge here and then could be handled in a large PR.

**Comment by jknndy** [comment]
> Something to try not to forget: this site seems to have both `ratings` and a `ratings_count` (ref: #1061).

That branch isn't merged into `main` yet so i'll open a cleanup PR once it is and I can add coverage to any newer scrapers.

Merging now

**Pull request merged**